Source/WebCore/platform/graphics/cg/ImageBufferCGBitmapBackend.cpp

```diff
@@ -49,7 +49,7 @@ size_t ImageBufferCGBitmapBackend::calculateMemoryCost(const Parameters& paramet
 
 std::unique_ptr<ImageBufferCGBitmapBackend> ImageBufferCGBitmapBackend::create(const Parameters& parameters, const ImageBufferCreationContext&)
 {
-    ASSERT(parameters.pixelFormat == ImageBufferPixelFormat::BGRA8);
+    ASSERT(parameters.bufferFormat.pixelFormat == ImageBufferPixelFormat::BGRA8);
```

Comment:
Looks like this change also needed to be made to ImageBufferCairoImageSurfaceBackend.cpp;
PS Debug is broken accordingly: https://build.webkit.org/#/builders/65/builds/54716

Reply:
@smfr Friendly reminder that this is still an issue.